src/components/Map3D/utilsVertex.ts

```diff
@@ -0,0 +1,265 @@
+/*
```

Comment:
I think this file could have some comments to make it easier for the rest of us to read the code.